gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -421,3 +504,15 @@ impl Stream for StreamProcessor<PayInvoiceResponse> {
         }
     }
 }
+
+impl Stream for StreamProcessor<CompleteHtlcsResponse> {
+    type Item = Result<CompleteHtlcsResponse, tonic::Status>;
```

Comment:
Is there any reason to not implement this generically: `impl<T> Stream for StreamProcessor<T>`? (ignoring if it's a good idea to do so)